ThingIFSDK/ThingIFSDK/OnboardWithVendorThingIDOptions.swift

```diff
@@ -41,3 +41,32 @@ public struct OnboardWithVendorThingIDOptions {
         self.layoutPosition = position
     }
 }
+
+extension OnboardWithVendorThingIDOptions: JsonObjectCompatible {
+
+    internal init(_ jsonObject: [String : Any]) throws {
+        // This method may not use so this method is not tested.
```

Comment:
@shinsuke-nara 
```
// This method may not use so this method is not tested.
```

I didn't expected to see comment like this.
If it is not tested, implementation is meaningless.

Reason why i unified them is not only a naming issue.
I thought that providing both and test both is a small effort and in most case useful.
If this is not true, let's divided them.